com.minres.coredsl/META-INF/MANIFEST.MF

```diff
@@ -14,7 +14,9 @@ Require-Bundle: org.eclipse.xtext,
  org.eclipse.xtext.util,
  org.eclipse.emf.common,
  org.antlr.runtime;bundle-version="[3.2.0,3.2.1)",
- org.eclipse.xtend.lib;bundle-version="2.14.0"
+ org.eclipse.xtend.lib;bundle-version="2.14.0",
+ com.google.guava,
```

Comment:
Quick nit, after the fact: Are you actually using Guava? Ctrl-F'ing here didn't turn up any results.